.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,38 @@
+on:
+  workflow_call:
+    inputs:
+      toolchain:
+        required: false
+        type: string
+        default: stable
+      coverage:
+        required: false
+        type: number
+        default: 14
+
+jobs:
+  coverage:
```

Comment:
**thought (blocking):** 
I suppose this job should be part of checks, right after test job ?